attestation-service/protos/attestation.proto

```diff
@@ -88,8 +89,17 @@ message SetPolicyRequest {
 }
 message SetPolicyResponse {}
 
+message ChallengeRequest {
+    Tee tee           = 1;
+    string tee_params = 2;
```

Comment:
We might need some refactor for the `tee_params`. It is marked `string` here, but the source could be a JSON map for better organization. So `map<string, string>` might be better somehow.

BTW I do not see any `tee_params` generated from the `attester` side. Can you give a link?

Reply:
Right, grpc and rest approach is new added in https://github.com/confidential-containers/trustee/pull/345/commits/3a6b84e8c8f6403a8801f5ed61ffd30d7f1bb3b0 according to @jialez0 's suggestion. IBM SE generates challenges directly.

Reply:
Recall this. See the comment in `attestation-service/attestation-service/src/bin/restful/mod.rs`

Reply:
Fixed, thanks!